content/en/news/security/istio-security-2021-001/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: ISTIO-SECURITY-2021-001
+subtitle: Security Bulletin
+description: JWT authentication can be bypassed when AuthorizationPolicy is misused.
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["1.9.0"]
+publishdate: 2021-03-1
```

Comment:
```suggestion
publishdate: 2021-03-01
```

Reply:
Hopefully this will fix the fact that it's not sorting correctly in the page list.